keyboards/hazel/chaz/promicro/info.json

```diff
@@ -0,0 +1,64 @@
+{
+    "manufacturer": "Hazel",
+    "keyboard_name": "Chaz",
+    "maintainer": "jasonhazel",
```

Comment:
`manufacturer`, `keyboard_name`, and `maintainer` lines are already specified in `info.json` in directory above, so they can be removed from this file.